### PR TITLE
fix(page-layout): adds custom styles to handle page layout

### PR DESCRIPTION
**Pull request by invincibleJai**
adds custom styles to handle page layout 

Tracks : https://jira.coreos.com/browse/ODC-677

Attached Gif with changes: 

page-bg

### CODE REVIEW

**Comment by invincibleJai** [comment]
@christianvogt PTAL, when you get some time

File: frontend/public/extend/devconsole/shared/components/EmptyState/EmptyState.scss [hunk 1]
```diff
@@ -1,10 +1,17 @@
-.odc-empty-state {
-  padding: 25px;
-  background: var(--pf-global--Color--light-200);
-  box-shadow: var(--pf-global--BoxShadow--inset);
+.odc-empty-section {
```
Comment (christianvogt) [inline frontend/public/extend/devconsole/shared/components/EmptyState/EmptyState.scss]:
```suggestion
.odc-empty-state {
```

File: frontend/public/extend/devconsole/shared/components/EmptyState/EmptyState.scss [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
 
-  &__card {
-    height: 100%;
-    text-align: center;
+  .odc-empty-state {
```
Comment (christianvogt) [inline frontend/public/extend/devconsole/shared/components/EmptyState/EmptyState.scss]:
```suggestion
  &__content {
```
Then move `&__card` so it's nested below the block.

**Review by christianvogt — COMMENTED** [review]
Fix the names of your style selectors.

File: frontend/public/extend/devconsole/shared/components/EmptyState/EmptyState.scss [hunk 3]
```diff
@@ -1,10 +1,17 @@
...
+    padding: 25px;
+    background: var(--pf-global--Color--light-200);
+    box-shadow: var(--pf-global--BoxShadow--inset);
```
Comment (christianvogt) [inline frontend/public/extend/devconsole/shared/components/EmptyState/EmptyState.scss]: PF4 designs also do not have a shadow. So let's remove it.

Comment (invincibleJai) [reply frontend/public/extend/devconsole/shared/components/EmptyState/EmptyState.scss]: +1

**Review by christianvogt — APPROVED** [review]

**Pull request merged**